administration/src/bp-modules/applications/ApplicationCard.tsx

```diff
@@ -240,22 +241,23 @@ const headerTypography: SxProps = {
 
 const ApplicationCard = ({
   application,
-  isSelectedForPrint,
   onDelete,
   onChange,
-  onPrintApplicationById,
 }: {
   application: Application
-  isSelectedForPrint: boolean
   onDelete: () => void
   onChange: (application: Application) => void
-  onPrintApplicationById: (applicationId: number) => void
 }) => {
   const { t } = useTranslation('applicationsOverview')
   const theme = useTheme()
   const config = useContext(ProjectConfigContext)
   const baseUrl = `${getApiBaseUrl()}/application/${config.projectId}/${application.id}`
   const appToaster = useAppToaster()
+  const printContentRef = useRef<HTMLDivElement>(null)
+  const printApplication = useReactToPrint({
+    contentRef: printContentRef,
+    pageStyle: applicationPrintViewPageStyle.styles,
+  })
```

Comment:
[Firefox issue] You can add `documentTitle: document.title,` so when you save the print it should have the name of the page instead of srcdoc.